pages/index.js

```diff
@@ -1,20 +1,19 @@
 import React from 'react';
 import Layout from '../components/Layout';
 import EventList from '../components/EventList';
-import EVENTS from '../fixtures/events'
+import { fixtureApi } from '../services';
```

Comment:
Because you have fixtures as the default in `services/index.js` I wonder if we can just say `import api from '../services'` and when we cut over it will not lead to any changes here.